src.ts/wallet/json-keystore.ts

```diff
@@ -96,7 +96,7 @@ function getAccount(data: any, _key: string): KeystoreAccount {
     const address = computeAddress(privateKey);
     if (data.address) {
         let check = data.address.toLowerCase();
-        if (check.substring(0, 2) !== "0x") { check = "0x" + check; }
+        if (check.startsWith("0x")) { check = "0x" + check; }
```

Comment:
This should be prefixed with `!`. But no worries, I caught it during merging. Just adding here for future observers. ;)

Thanks! :)